tigerpath/majors_and_certificates/majors/CEE.json

```diff
@@ -41,14 +41,23 @@
               "min_needed": null,
               "explanation": null,
               "course_list": [
-                "CEE 205: Mechanics of Solids",
-                "CEE 262A: Structures and the Urban Environment",
```

Comment:
https://ua.princeton.edu/academic-units/department-civil-and-environmental-engineering